tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -29,7 +29,7 @@ Feature: full text search
       | spaces           |
 
 
-  Scenario Outline: sharee searches a file tagged by sharer using tag
+  Scenario Outline: sharee tags another file from a shared resource using the same tag as the sharer
```

Comment:
This is not a tagging scenario. Instead, it is for searching using tags. Please, rename as in this comment https://github.com/owncloud/ocis/pull/6659#discussion_r1251540695
```suggestion
  Scenario Outline: sharee searches shared files using a tag
```